g711.py

```diff
@@ -0,0 +1,107 @@
+__all__ = ['alaw_encode', 'alaw_decode', 'ulaw_encode', 'ulaw_decode', 'g711_read', 'g711_write', 'open']
+
+from _g711.lib import alaw_encode as _alaw_encode, alaw_decode as _alaw_decode, ulaw_encode as _ulaw_encode, ulaw_decode as _ulaw_decode
+
+import builtins
+
+from cffi import FFI
+ffi = FFI()
+
+
+class Error(Exception):
```

Comment:
I think we can just use the inbuilt exceptions here and get rid of this custom exception call entirely.